src/engine/sysDef.cpp

```diff
@@ -144,6 +144,8 @@ DivSystem DivEngine::systemFromFileFur(unsigned char val) {
       return DIV_SYSTEM_YM2610B_EXT;
     case 0xe0:
       return DIV_SYSTEM_QSOUND;
+    case 0xfe:
+      return DIV_SYSTEM_OPL4;
```

Comment:
The system IDs for OPL4 and MultiPCM are defined already in papers/format.md
Do not use 0xfe. It was used for Realtek HD Audio (or I think Kontakt 5?) in the April Fools' version.

0xae - OPL4
0xaf - OPL4 (drums)